packages/@react-spectrum/s2/stories/ActionButton.stories.tsx

```diff
@@ -224,16 +224,18 @@ NotificationBadges.parameters = {
   docs: {
     source: {
       transform: () => {
-        return `let badgeValue = 10;
-        let formattedValue = useNumberFormatter().format(badgeValue);
-      
-        return (
-          <div style={{display: 'flex', gap: 8, padding: 8, justifyContent: 'center'}}>
-            <ActionButton aria-label="Messages has new activity" {...props}><CommentIcon /><NotificationBadge /></ActionButton>
-            <ActionButton aria-label={\`\${formattedValue} notifications\`}  {...props}><BellIcon /><NotificationBadge value={badgeValue} /></ActionButton>
-            <ActionButton {...props}><CommentIcon /><Text>Messages</Text><NotificationBadge value={5} /></ActionButton>
-            <ActionButton {...props}><Text>Notifications</Text><NotificationBadge value={105} /></ActionButton>
-          </div>
+        return `
+let badgeValue = 10;
+let formattedValue = useNumberFormatter().format(badgeValue);
+
+return (
+  <div style={{display: 'flex', gap: 8, padding: 8, justifyContent: 'center'}}>
+    <ActionButton aria-label="Messages has new activity"><CommentIcon /><NotificationBadge /></ActionButton>
+    <ActionButton aria-label={\`\${formattedValue} notifications\`} ><BellIcon /><NotificationBadge value={badgeValue} /></ActionButton>
+    <ActionButton><CommentIcon /><Text>Messages</Text><NotificationBadge value={5} /></ActionButton>
+    {/* cannot have an isQuiet label-only Action Button with a Notification Badge */}
```

Comment:
i added a comment here about the label-only quiet action button but i can also make it so that the story example doesn't change to isQuiet when it is label-only

Reply:
or hide that button when isQuiet?